sematic/config/settings.py

```diff
@@ -42,6 +41,18 @@ class ProfileSettings:
     settings: PluginsSettings = field(default_factory=dict)
 
 
+_DEFAULT_PROFILE = "default"
+_DEFAULT_PROFILE_SETTINGS = ProfileSettings(
+    scopes={},
+    settings={
+        # providing textual path instead of importing in order to avoid a dependency cycle
+        # these plugins are "known" and will always be part of the server deployment
+        "sematic.config.server_settings.ServerSettings": {},
+        "sematic.config.user_settings.UserSettings": {},
```

Comment:
I wonder if there's a way we can be robust against a user who *has* a `settings.yaml` file already present, but it doesn't contain `sematic.config.user_settings.UserSettings`. Would that be difficult to implement?

Reply:
In `get_active_settings`, we could test if `sematic.config.user_settings.UserSettings` is present in `profile_settings.settings` and if not, add it. Then we wouldn't need the change above.

Reply:
Excellent points! Done, and added tests.